Tasks/Gradle/Gradle.ps1

```diff
@@ -76,6 +88,33 @@ if ($jdkPath)
     Write-Verbose "JAVA_HOME set to $env:JAVA_HOME"
 }
 
+$buildRootPath = Split-Path $wrapperScript -Parent
+$reportDirectoryName = "CodeCoverage"
+$reportDirectory = Join-Path $buildRootPath $reportDirectoryName
+
+if(Test-Path $reportDirectory)
+{
+   # delete any previous code coverage data 
+   rm -r $reportDirectory -force | Out-Null
```

Comment:
Again may not be common but delete directory errors are being ignored which could result in unpredictable results uploaded during the build.